src/site/cpp-arrays.rst

```diff
@@ -0,0 +1,47 @@
+*************************
+C++ Lessons #3 --- Arrays
+*************************
+
+.. warning::
+
+    * For these, I will only go over the C++ details at a high level
+    * Getting into the nitty gritty of C++ is not the purpose of this aside
+    * Some of the concepts require knowledge of more advanced ideas that have not been covered yet
+        * If this happens, do not panic
+        * it probably makes more sense to revisit this later in the course
+    * For simplicity, some of the examples will use less than ideal implementations
+
+
+.. note::
+
+    Here we will go over the basics of arrays in C/C++; however, in practice you will likely be making use of C++'s
+    standard library for your arrays.
+
+        * `std::array <https://en.cppreference.com/w/cpp/container/array>`_
+        * `std::vector <https://en.cppreference.com/w/cpp/container/vector/>`_
```

Comment:
A vector isn't really an array, it's a vector. It's definitely a good replacement for c-style arrays though -- probably better than `std::array` honestly. I'd probably mention `std::array` in passing, and push more for learning `std::vector`.

Reply:
I agree --- will massage it.